firmware/integration/rusefi_config.txt

```diff
@@ -1098,7 +1098,7 @@ int16_t tps2Max;;"ADC", 1, 0, 0, 1023, 0
 	custom InjectionTimingMode 1 bits, U08, @OFFSET@, [0:1], "End of injection", "Start of injection", "Center of injection"
 	InjectionTimingMode injectionTimingMode;Sets what part of injection's is controlled by the injection phase table.
 
-	debug_mode_e debugMode;See http://rusefi.com/s/debugmode
+	debug_mode_e debugMode;See https://wiki.fome.tech/r/debugmode
```

Comment:
Q: What provides this `r/debugmode` feature of the Wiki site (it redirected me to https://wiki.fome.tech/Advanced-Features/Debug-Mode/ -- cool)?

Reply:
Oh I see, thanks!

https://github.com/FOME-Tech/fome-fw/issues/235#issuecomment-1715299377

Reply:
I've also left some info in the README about this feature: https://github.com/FOME-Tech/wiki?tab=readme-ov-file#redirection

Reply:
> info in the README about this feature: https://github.com/FOME-Tech/wiki?tab=readme-ov-file#redirection

this was exactly the place where I found this info :)